README.md

```diff
@@ -75,7 +72,7 @@ GitHub Actions CI will run all tests and build commands per package manager on e
 
 ### Local Testing
 
-TBD until development progresses. We will use Rake, Fastlane, or some other tool for easy command-line build tasks.
+We use [Fastlane](https://docs.fastlane.tools/getting-started/ios/setup/) for command-line build tasks.
```

Comment:
We really should have instructions for how to use fastlane to run our tests locally. This can be it's own ticket.